content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
```

Comment:
<p>The plots are great, I think that closes the tutorial nicely!</p><p> </p><p>Tip: you can add <code>plt.tight_layout()</code>  to improve on the spacing between the subplots - at least locally I had the title and x-axis label of the subplots superimposed without using <code>tight_layout</code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
That's a good idea. Let me try a side-by-side (left-to-right) layout with `subplots`.